torch/_export/serde/serialize.py

```diff
@@ -939,6 +948,57 @@ def output_node_at_index(node, index):
 
         return output_arguments
 
+    def serialize_arbitrary_outputs(self, node: torch.fx.Node) -> List[Argument]:
```

Comment:
"arbitrary" doesn't seem right, given that we do not accept arbitrary outputs. Maybe just `serialize_hop_outputs` or whatever